webapp/sources/rudder/rudder-web/src/main/elm/healthcheck/sources/View.elm

```diff
@@ -87,8 +87,8 @@ displayBigMessage checks =
     [
         severityLevelToIcon level
       , case level of
-          Critical    -> text "Critical error"
-          Warning     -> text "Should be improved"
+          Critical    -> text "Some elements require your attention"
+          Warning     -> text "Some elements require your attention"
```

Comment:
Same message for Critical and Warning ?

Reply:
yes, there is a lot of other visual clue for the difference (the categorie, color, etc)